modules/samtools/view/meta.yml

```diff
@@ -39,23 +39,16 @@ output:
       description: |
         Groovy Map containing sample information
         e.g. [ id:'test', single_end:false ]
-  - bam:
+  - output:
       type: file
-      description: filtered/converted BAM/SAM file
-      pattern: "*.{bam,sam}"
-  - cram:
-      type: file
-      description: filtered/converted CRAM file
-      pattern: "*.cram"
+      description: filtered/converted BAM/SAM/CRAM file
+      pattern: "*.{bam,sam,cram}"
   - versions:
```

Comment:
These are three separate output channels according to the module `main.nf`

Reply:
updated